CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+### 74.1.2-beta.1 [#1674](https://github.com/openfisca/openfisca-france/pull/1674)
```

Comment:
S'il n'y a pas de changement fonctionnel, pas besoin de publier une nouvelle version 🙂

Reply:
Oui, tout à fait. Mon but ici est plus de tester for failure. Le dernier commit avant le merge va tout nettoyer.

Reply:
Par contre, j'ai une erreur sur le [`check-version-number`](https://github.com/openfisca/openfisca-france/runs/3686866666?check_suite_focus=true). Je pense que je ne peux pas faire un retour en arrière une fois le fichiers modifiés 🤔